gui/src/store/modules/user-login.ts

```diff
@@ -35,6 +35,7 @@ Vue.use(Vuex);
 export default class UserLogin extends VuexModule {
   public user: any = {};
   public userPermissions: object = {};
+  public unsupportedFeatures: object = {};
```

Comment:
we should avoid to use object type. I know this was done in past but this should be removed. We can define our own class and should use that class to create object

Reply:
Created new interface and added it everywhere unsupportedFeatures used.